docs/en-US/component/pagination.md

```diff
@@ -74,7 +74,7 @@ pagination/more-elements
 | pager-count                         | number of pagers. Pagination collapses when the total page count exceeds this value                                             | ^[number]`5 \| 7 \| 9 \| 11 \| 13 \| 15 \| 17 \| 19 \| 21`                        | 7                                    |
 | current-page / v-model:current-page | current page number                                                                                                             | ^[number]                                                                         | 1                                    |
 | default-current-page                | default initial value of current-page                                                                                           | ^[number]                                                                         | -                                    |
-| layout                              | layout of Pagination, elements separated with a comma                                                                           | ^[string]`string (consists of sizes, prev, pager, next, jumper, ->, total, slot)` | prev, pager, next, jumper, ->, total |
+| layout                              | layout of Pagination, elements separated with a comma                                                                           | ^[string]`string (consists of sizes, prev, pager, next, jumper, ->, total, slot)` | prev, sizes, pager, next, jumper, ->, total |
```

Comment:
## Description

```ts
export const paginationProps = buildProps({
  layout: {
    type: String,
    default: (
      ['prev', 'pager', 'next', 'jumper', '->', 'total'] as LayoutKey[]
    ).join(', '),
  },
});
```

Thank your contribution.

but props default information matches the documentation information, so no changes are required.

Reply:
was my fault, didn't notice that was the default value, I thought it was an optional parameter